spec/pmf_nfpse_generator_spec.rb

```diff
@@ -101,13 +92,17 @@
 
        it "invalid cpf_cnpj" do
          lib.cpf_cnpj = "000"
-         expect(lib.to_xml!).to be_falsey
+         VCR.use_cassette('to_xml_invalid_cpf_cnpj') do
+           expect(lib.to_xml!).to be_falsey
```

Comment:
Agreed! We are not raising exceptions anymore, so we should change the method name to just `to_xml`